assets/js/amp-story-editor-blocks.js

```diff
@@ -425,7 +451,35 @@ var ampStoryEditorBlocks = ( function() { // eslint-disable-line no-unused-vars
 				PanelBody = wp.components.PanelBody,
 				SelectControl = wp.components.SelectControl,
 				parentClientId = select.getBlockRootClientId( props.clientId ),
-				parentBlock;
+				parentBlock = select.getBlock( parentClientId );
+
+			if ( 'core/image' === name && ( parentBlock && 'amp/amp-story-grid-layer-background-image' !== parentBlock.name ) ) {
+				const ToggleControl = wp.components.ToggleControl,
+					ampShowImageCaption = !! attributes.ampShowImageCaption;
+				inspectorControls = el( InspectorControls, { key: 'inspector' },
+					el( PanelBody, { title: __( 'Toggle Image Caption', 'amp' ), key: 'amp-image-no-caption' },
```

Comment:
@jacobschweitzer Do you think it might make sense to move this under AMP Story Setting section as well? To make it clear for the editor that this setting is related to AMP Story only and added with the functionality of AMP Story. Thoughts?

Reply:
Yeah that makes sense, I've moved it 👍